app/controllers/admin/shelters_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::SheltersController < ApplicationController
+
+    def index
+        @admin_shelters = Shelter.reverse_alphabetical_order
+
+        @shelters_with_pending_apps = Shelter.sort_by_pending
```

Comment:
Great work encapsulating the querying at the model level and calling those methods here! Very MVC compliant